dts/bindings/pinctrl/microchip,xec-pinctrl.yaml

```diff
@@ -0,0 +1,116 @@
+# Copyright (c) 2020 Linaro Limited
+# Copyright (c) 2021 Microchip Technology Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+    Microchip XEC Pin controller Node
+    Based on pincfg-node.yaml binding.
+    The MCHP XEC pin controller is a singleton node responsible for controlling
+    pin function selection and pin properties. For example, you can use this
+    node to select peripheral pin functions. For example, I2C
```

Comment:
`For example, I2C`....? seems like it's missing content

Reply:
that is extraneous. will remove `For example, I2C`